.github/workflows/main.yaml

```diff
@@ -83,11 +73,15 @@ jobs:
           path: ~/go/pkg/mod
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
 
+      - name: Install dependences
+        run: sudo apt-get install -y upx-ucl gcc-aarch64-linux-gnu libc6-dev-arm64-cross gcc-arm-linux-gnueabi libc6-dev-armel-cross
+
+      - name: Install golangci-lint
+        run: curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.39.0
```

Comment:
Why Install golangci-lint when making Multiple docker image build?

Reply:
Because in the building process, it will run all steps including verify, lint, build.

Reply:
So it's not reasonable to run the lint when make build.